sal-common/src/main/java/org/ow2/proactive/sal/model/Cloud.java

```diff
@@ -6,66 +6,100 @@
 package org.ow2.proactive.sal.model;
 
 import java.io.Serializable;
-import java.util.Locale;
-import java.util.Objects;
+import java.util.LinkedHashMap;
+import java.util.Map;
 
 import javax.persistence.*;
 
+import org.ow2.proactive.sal.util.ModelUtils;
+
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonValue;
 
 import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 
 /**
  * Representation of a cloud used by Cloudiator
  */
-@AllArgsConstructor
+@Getter
+@Setter
```

Comment:
A small comment here: For getters is not a big deal but we might not want setters for all fields to avoid future misuse. If a setter is not required for all fields, please use `@Setter` directly on the needed fields.